scripts/load_cassandra.sh

```diff
@@ -1,20 +1,32 @@
 #!/bin/bash
-EXE_DIR=${EXE_DIR:-$(dirname $0)}
+
+# Ensure loader is available
+EXE_FILE_NAME=${EXE_FILE_NAME:-$(which tsbs_load_cassandra)}
+if [[ -z "$EXE_FILE_NAME" ]]; then
+    echo "tsbs_load_cassandra not available. It is not specified explicitly and not found in \$PATH"
+    exit 1
+fi
+
+# Load parameters - common
 DATA_FILE_NAME=${DATA_FILE_NAME:-cassandra-data.gz}
+DATABASE_PORT=${DATABASE_PORT:-9042}
+
+# Load parameters - personal
 PROGRESS_INTERVAL=${PROGRESS_INTERVAL:-20s}
```

Comment:
Actually we can add PROGRESS_INTERVAL to the common file, it should be `10s` for all of them.

Reply:
Sounds reasonable, I'll move PROGRESS_INTERVAL to common file and change to 10 sec